api/meter.go

```diff
@@ -35,13 +35,20 @@ type RelayMeter interface {
 	// LoadBalancerRelays returns the metrics for an Endpoint, AKA loadbalancer
 	LoadBalancerRelays(endpoint string, from, to time.Time) (LoadBalancerRelaysResponse, error)
 	AllLoadBalancersRelays(from, to time.Time) ([]LoadBalancerRelaysResponse, error)
+	AppLatency(app string) (AppLatencyResponse, error)
+	AllAppsLatency() ([]AppLatencyResponse, error)
```

Comment:
Perhaps `AllLatencies` or `AllAppsLatencies` or `Latencies` are better candidates for the name of this method?

Reply:
Updated.